### DISCUSSION TITLE
Image location best practices?

### DISCUSSION THREAD

**Discussion by ngoodroe** [binokochumolvarghese/lightbi-hugo#78] [General]
Love the theme! Been playing around with it all day learning Hugo and I had a question about images and where I need to place them. For the main page, in order to get my "thumbnail" (the image referenced in the header at the start of every post), I have to have it in assets, but to have an image inside of my post, I have to have it in the static folder. 
Is there any way that I can have all my images in one place and be able to call them for both uses?

**Comment by binokochumolvarghese** [comment]
yes, you can save all the images in the asset folder and can call the same as below in the markdown.

```
{{< figure src="/img/folder/image.jpg" width="" caption="" attrlink="" >}}
````
